frame/support/test/tests/derive_no_bound_ui/eq.stderr

```diff
@@ -4,4 +4,9 @@ error[E0277]: can't compare `Foo<T>` with `Foo<T>`
 6   | struct Foo<T: Config> {
     |        ^^^ no implementation for `Foo<T> == Foo<T>`
     |
-    = help: the trait `std::cmp::PartialEq` is not implemented for `Foo<T>`
+   ::: $RUST/core/src/cmp.rs
+    |
+    | pub trait Eq: PartialEq<Self> {
+    |               --------------- required by this bound in `Eq`
+    |
```

Comment:
CI doesn't show this part of the error. it might be due to CI not having rust-src component or something like this.
```suggestion
```

Reply:
No. CI is still using the old image. It works with the new image that is not yet enabled.